README.md

```diff
@@ -30,6 +53,16 @@ This plugin is in Beta. Live support is available for the latest builds through
 - **Optional:**
    - [Nucleus](https://nucleuspowered.org) - 1.0.1-LTS+
 
+## 依赖插件
+
+- **必须插件**
+   - [Sponge](https://www.spongepowered.org/downloads) - 1.10.2-2281-5.2.0-BETA-2519+
+   - [Grief Prevention](https://forums.spongepowered.org/t/griefprevention-official-thread/1123) - 1.10.2-4.0.0.332+
+   - Permission Plugin - 推荐使用[LuckPerms](https://forums.spongepowered.org/t/luckperms-an-advanced-permissions-plugin/14274)
+- **可选插件**（译者注：这里多推荐一个经济类插件，以便支持岛屿的经济扩展，仅支持API5.0/6.0 here give one econ plugin for sport economy,only for API5.0/6.0）
+   - [Nucleus](https://nucleuspowered.org) - 1.0.1-LTS+
+   - [Total Economy](https://ore.spongepowered.org/Erigitic/Total-Economy)
```

Comment:
Why was this added to the list? I don't even have an Economy support at this time.

Reply:
- [X] Expanding islands using claim blocks or other resource (econ)
this line? didnt it means your support Economy ?

Reply:
I haven't gotten around to implementing that ha ha...

Reply:
Only claim blocks are supported at this time.

Reply:
well...→_→!!

Reply:
I'm a slacker, I know... I'll fix it on the English version to make it known that the Economy support is upcoming.

Reply:
don't update too quickly,i'm slacker too.

Reply:
Too late ⏩

Reply:
will, your means i need retranslation again? GG...in Chinese it is 11pm now...

Reply:
There's no rush!